kubectl-rook-ceph.sh

```diff
@@ -263,6 +265,54 @@ function run_purge_osd() {
     rook ceph osd remove --osd-ids=$1 --force-osd-removal=$force_removal"
 }
 
+####################################################################################################
+# 'kubectl rook-ceph debug commands
+####################################################################################################
+
+function run_create_debug(){
+  [[ -z "${1:-""}" ]] && fail_error "Missing 'mon/osd/mgr/mds' deployment"
+  deployment_name=$1
+  echo "create debug mode for $deployment_name"
+
+  if output=$(TOP_LEVEL_COMMAND --namespace "$ROOK_OPERATOR_NAMESPACE" get deployment "$deployment_name"); then
+    echo "deployment $deployment_name exist, $output" 
+  else  
+    echo "deployment $deployment_name doesn't exist"
+    exit 1
+  fi
+  
+  # copy the deployment spec before scaling it down
+  deployment_spec=$(TOP_LEVEL_COMMAND get  deployments --namespace "$ROOK_OPERATOR_NAMESPACE"  "$deployment_name"   -o json | jq -r ".spec")
+  
+  # scale the deployment to 0
+  TOP_LEVEL_COMMAND scale deployments --namespace "$ROOK_OPERATOR_NAMESPACE" "$deployment_name" --replicas=0
+  
+  # (Refrences: https://kubernetes.io/docs/tasks/debug/debug-application/debug-running-pod/#copying-a-pod-while-changing-its-command)
+  # kubectl debug rook-ceph-mon-d-7947b7c46c-2c98h -it --copy-to=rook-ceph-mon-d-debug --image=quay.io/ceph/ceph:v17.2.1 -nrook-ceph 
+  # create debug deployment
+  cat <<eof | TOP_LEVEL_COMMAND create -f -
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+        name:"$deployment_name-debug"
+        namespace: "$ROOK_OPERATOR_NAMESPACE"
+        labels:
+            ceph.rook.io/debug: "true"
+    spec:
+        $deployment_spec
```

Comment:
The thing to look ,
How to edit this var `$deployment_spec`  for changing the cmd